src/libs/sfdk/buildengine.cpp

```diff
@@ -950,6 +954,47 @@ QString BuildEngineManager::installDir()
     return s_instance->m_installDir;
 }
 
+QString BuildEngineManager::defaultBuildHostName()
+{
+    if (!s_instance->m_defaultBuildHostName.isNull())
+        return s_instance->m_defaultBuildHostName;
+
+    // A non-blocking lookup was initiated during initialization, but it takes
+    // longer than anticipated - let's be patient.
+
+    QTime time;
```

Comment:
time is only used for debug message. I wonder if it's possible to have it instantiated only if debug messages are enabled?

Reply:
That would be clearly a premature optimization :)